lm_eval/base.py

```diff
@@ -740,6 +741,9 @@ def process_results(self, doc, results):
                     rouge_scores = utils.flatten(rouge_scores)
                     # Merge all the rouge-type scores into the `out` dict.
                     out = {**out, **rouge_scores}
+                elif metric == "SARI":
+                    text = self.doc_to_text(doc)
+                    out = {"sari": metrics.sari(text, pred, target)}
```

Comment:
Is `text` okay here?

Reply:
@cjlovering , No I don't think so, as text now has the prompt applied. The prompt (esp. if long) is not what SARI is measuring.